### PR TITLE
changed: add helper template to serialize and broadcast static types

### PR DESCRIPTION
**Pull request by akva2**
4 uses, simpler syntax in the problem code. i think this justifies a helper template.

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this please

**Comment by akva2** [comment]
okay to merge?

File: ebos/eclmpiserializer.hh [hunk 1]
```diff
@@ -27,7 +27,7 @@ namespace Opm {
 
 class EclMpiSerializer {
 public:
-    EclMpiSerializer(Dune::MPIHelper::MPICommunicator comm) :
+    EclMpiSerializer(Dune::CollectiveCommunication<Dune::MPIHelper::MPICommunicator> comm) :
```
Comment (bska) [inline ebos/eclmpiserializer.hh]: Is there a reason that this constructor is not marked `explicit`?

Comment (akva2) [reply ebos/eclmpiserializer.hh]: no, it should be. i'll include that in upcoming PR that depend on this, so i don't have to rebase this one.

**Pull request merged**